db/seeds.rb

```diff
@@ -2,7 +2,7 @@
 
 def make_books
   SEEDS[:books].each do |book|
-    Book.create!  book
+    Book.create!  book.merge!(in_store: true)
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.